crates/swc_ecma_transforms_compat/src/es2017/async_to_generator.rs

```diff
@@ -804,7 +813,7 @@ impl Actual {
         let right = make_fn_ref(
             FnExpr {
                 ident: None,
-                function: f,
+                function: f.take(),
```

Comment:
Same here

Reply:
seems like I can't move Function in here?